assignments/closure.js

```diff
@@ -1,14 +1,27 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
 
+function dateOfBirth(month, day, year){
```

Comment:
This is a great start. I noticed your console.log is a bit off and you're not using myBirth, though. Here's how it looks with those fixed:
```
function dateOfBirth(month, day, year){
  const myBirth = "The date of my birth is:";

  function myBirthday (){
    return `${myBirth} ${month} ${day}, ${year}`;
  }

  return myBirthday();
}

console.log(dateOfBirth("May", "15", '1993'));
```